src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
+    protected DateAndTime at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = new DateAndTime(at);
+    }
+
+    @Override
+    public String toSave() {
+        return "E | " + (isDone ? 1 : 0) + " | " + description + " | " + at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at + ")";
```

Comment:
Calling the super class toString() here is pretty smart. I've never thought of it.

Reply:
Thanks :)